iOS/DuckDuckGo/Refactoring/Default/DefaultOmniBarView.swift

```diff
@@ -138,10 +143,19 @@ class DefaultOmniBarView: UIView {
         settingsButton.addGestureRecognizer(longPressGesture)
     }
 
-    private func configureShareLongPressButton() {
-        let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(handleShareLongPress(_:)))
-        longPressGesture.minimumPressDuration = 0.7
-        accessoryButton.addGestureRecognizer(longPressGesture)
+    private func configureAccessoryLongPressButton() {
+
+        if state.dependencies.featureFlagger.isFeatureOn(.customizableActionButton) &&
```

Comment:
Should it also check for internal user flag?

Reply:
The feature flag is driven by `isInternalOnly` thing and seems to be working as intended.